data/campaigns/Under_the_Burning_Suns/units/quenoth/Dustbok.cfg

```diff
@@ -4,7 +4,6 @@
     id=Quenoth Dustbok
     name= _ "Dustbok"
```

Comment:
This seems like the ideal place for a `#po:` comment explaining the word so that translators can reasonably make up a word for the translations.